src/components/Elements/Chevron.jsx

```diff
@@ -0,0 +1,5 @@
+import React from 'react'
+
+const Chevron = () => <svg xmlns="http://www.w3.org/2000/svg" width="12" height="12" className="chevron" viewBox="0 0 185.344 185.344"><path d="M92.672 144.373c-2.752 0-5.493-1.044-7.593-3.138L3.144 59.3c-4.194-4.198-4.194-10.99 0-15.18 4.194-4.198 10.987-4.198 15.18 0l74.347 74.342 74.347-74.34c4.193-4.2 10.986-4.2 15.18 0 4.193 4.193 4.193 10.98 0 15.18l-81.94 81.933c-2.094 2.094-4.84 3.138-7.588 3.138z" /></svg>
```

Comment:
In fact, I see no reason to have a Component for this rather than just putting it in the CSS (in `track-key.scss`) as per https://github.com/JSainsburyPLC/react-timelines/blob/master/src/scss/components/_track-key.scss#L97